src/codegen/sys/lib_.rs

```diff
@@ -106,12 +107,12 @@ fn generate_bitfields<W: Write>(w: &mut W, items: &[&Bitfield])
     try!(writeln!(w, ""));
     for item in items {
         try!(writeln!(w, "bitflags! {{\n{}#[repr(C)]\n{0}flags {}: c_uint {{",
-                      tabs(1), item.c_type));
+                      TAB, item.c_type));
         for member in &item.members {
             let val: i64 = member.value.parse().unwrap();
-            try!(writeln!(w, "{}const {} = {},", tabs(2), member.c_identifier, val as u32));
+            try!(writeln!(w, "{}{0}const {} = {},", TAB, member.c_identifier, val as u32));
```

Comment:
To me it would silghtly improve readability even when there's just one tab but that would be a lot of work for little gain.

We could just use `"\t"` and replace it with `"    "` in a writer adapter. Many editors will highlight `\t` in a string.

```rust
try!(writeln!(w, "\t\tconst {} = {},", member.c_identifier, val as u32));
```